.github/workflows/auto_request_review.yml

```diff
@@ -1,5 +1,7 @@
 name: Auto Request Review
 
+permissions: read-all
+
```

Comment:
It looks like the `read-all` permission is specific to `GITHUB_TOKEN`:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview

Better to remove it here so there's no confusion?

Reply:
We've typically set it to read-all as the default at the top level of the action. Without the system generates high token-permission warnings in the security tab. If we add anything else to this that doesn't need the custom token we'll already be set and can override at the job level.

Reply:
SGTM